README.org

```diff
@@ -1,6 +1,9 @@
 #+author: Cam Parra
 
-* Sovrin Ledger Payment plugins
+#+ATTR_HTML: :style margin-left: auto; margin-right: auto;
+[[./banner.png]]
```

Comment:
what is banner.png?

Reply:
https://github.com/sovrin-foundation/sovrin/blob/master/banner.png